abacustest/lib_model/model_006_FDForce.py

```
@@ -432,64 +432,68 @@ def plot_force(self, results):
+            try:
+                distance = results[case]["r1"]["pos"]
+                energy = results[case]["r1"]["energy"]
+                force = results[case]["r1"]["force"]
+                fd_force = results[case]["r1"]["fd_force"]
+
+                d,e,f,fd = comm.clean_none_list(distance,energy,force,fd_force)
+                x, y = comm.clean_none_list(results[case]["r2"]["step"],results[case]["r2"]["fd_force"])
+                if len(d) == 0 or len(x) == 0:
+                    continue
+                
+                fig,ax = plt.subplots(1,2,figsize=(12,5))
```

Ruff (0.14.10)

Unpacked variable `fig` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)